proposals/csharp-9.0/native-integers.md

```diff
@@ -218,6 +218,18 @@ class B : A
 }
 ```
 
+### Binary numeric promotions
+The _binary numeric promotions_ informative text (see [§11.4.7.3](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/expressions.md#11473-binary-numeric-promotions)) in C# spec) is updated as follows:
+
+> -   …
+> -   Otherwise, if either operand is of type `ulong`, the other operand is converted to type `ulong`, or a binding-time error occurs if the other operand is of `type sbyte`, `short`, `int`, **`nint`**, or `long`.
```

Comment:
Should `nuint` also be a binding-time error? What if on that (future) platform, `nuint` is larger than the `ulong`?

Reply:
> What if on that (future) platform, `nuint` is larger than the `ulong`?

According to [ECMA-335](https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf) (see II.7.1 Types), the native int type is "32- or 64-bit signed integer whose size is platform-specific".